src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationHandler.cs

```diff
@@ -270,36 +289,13 @@ private CookieOptions BuildCookieOptions()
 
                 Options.Notifications.ResponseSignedIn(signedInContext);
 
-                Response.Headers.Set(
-                    HeaderNameCacheControl,
-                    HeaderValueNoCache);
-
-                Response.Headers.Set(
-                    HeaderNamePragma,
-                    HeaderValueNoCache);
-
-                Response.Headers.Set(
-                    HeaderNameExpires,
-                    HeaderValueMinusOne);
-
                 var shouldLoginRedirect = Options.LoginPath.HasValue && Request.Path == Options.LoginPath;
```

Comment:
Request.Path may be wrong (altered by Map) when SignIn is applied immediately. Do we need to add base.OriginalPath like we have for OriginalPathBase?